scripts/jinx.lic

```diff
@@ -355,7 +355,7 @@ module Jinx
         name = File.basename(asset.fetch(:file))
         repo_name = repo.fetch(:name)
         Log.out("updating %s file metadata for %s from %s" % [asset[:type], name, repo_name], label: %i(metadata update))
-        create({name: name, repo: repo_name, digest: asset.fetch(:md5)})
+        create(**{name: name, repo: repo_name, digest: asset.fetch(:md5)})
```

Comment:
:+1: I should upgrade from 2.6 to 2.7 so that I get deprecation warnings when I forget to do this.